src/DssSpell.sol

```diff
@@ -58,62 +55,145 @@ contract DssSpellAction is DssAction {
+        DssExecLib.setIlkStabilityFee("ETH-A", FIVE_PT_TWO_FIVE_PCT_RATE, /* doDrip = */ true);
+
+        // Increase the ETH-B Stability Fee (SF) by 1.55%, from 4.20% to 5.75%.
+        DssExecLib.setIlkStabilityFee("ETH-B", FIVE_PT_SEVEN_FIVE_PCT_RATE, /* doDrip = */ true);
+
+        // Increase the ETH-C Stability Fee (SF) by 1.55%, from 3.45% to 5.00%.
+        DssExecLib.setIlkStabilityFee("ETH-C", FIVE_PCT_RATE, /* doDrip = */ true);
+
+        // Increase WBTC-A Stability Fee (SF) by 0.06%, from 5.8% to 5.86%
+        DssExecLib.setIlkStabilityFee("WBTC-A", FIVE_PT_EIGHT_SIX_PCT_RATE, /* doDrip = */ true);
+
+        // Increase WBTC-B Stability Fee (SF) by 0.06%, from 6.3% to 6.36%
+        DssExecLib.setIlkStabilityFee("WBTC-B", SIX_PT_THREE_SIX_PCT_RATE, /* doDrip = */ true);
+
+        // Increase WBTC-C Stability Fee (SF) by 0.06%, from 5.55% to 5.61%
+        DssExecLib.setIlkStabilityFee("WBTC-C", FIVE_PT_SIX_ONE_PCT_RATE, /* doDrip = */ true);
+
 
-    // ---------- Trigger Spark Proxy Spell ----------
-    // address internal constant SPARK_SPELL = address(0);
+        // ---------- Initial RETH-A Offboarding  ----------
+        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-6/22231
 
-    function _updateDoc(bytes32 ilk, string memory doc) internal {
-        ( , address pip, uint48 tau, ) = RwaLiquidationOracleLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
-        require(pip != address(0), "DssSpell/unexisting-rwa-ilk");
+        // Set DC to 0.
+        (,,,uint256 line,) = VatLike(MCD_VAT).ilks("RETH-A");
+        DssExecLib.removeIlkFromAutoLine("RETH-A");
+        DssExecLib.setValue(MCD_VAT, "RETH-A", "line", 0);
+        // NOTE: decreasing globalline suniglow level API because of precision loss when using DssExec
+        DssExecLib.setValue(MCD_VAT, "Line", VatLike(MCD_VAT).Line() - line);
 
-        // Init the RwaLiquidationOracle to reset the doc
-        RwaLiquidationOracleLike(MIP21_LIQUIDATION_ORACLE).init(
-            ilk, // ilk to update
-            0,   // price ignored if init() has already been called
-            doc, // new legal document
-            tau  // old tau value
+
+        // ---------- Reconfiguring Andromeda RWA015-A  ----------
+        // Forum: https://forum.makerdao.com/t/poll-request-reconfiguring-rwa-allocator-vaults/22159
+        // Poll: https://vote.makerdao.com/polling/QmPoLbah
+
+        // Set the Maximum Debt Ceiling (line) to 3 billion DAI.
+        DssExecLib.setIlkAutoLineDebtCeiling("RWA015-A", 3 * BILLION);
+
+        // Bump Oracle price to account for new DC and SF
+        // NOTE: the formula is `Debt ceiling * [ (1 + RWA stability fee ) ^ (minimum deal duration in years) ] * liquidation ratio`
+        // NOTE: `minimum deal duration in year` is not used is that formula as stability fee is 0
+        // bc -l <<< 'scale=18; 3000000000 * e(l(0) * (1)) * 1.00' | cast --to-wei
```

Comment:
should probably be a explanation line instead (since SF is 0, the value will be the same as line)

like [here](https://github.com/makerdao/spells-mainnet/blob/04771b7a1ea416b66b75784d205fc1bbc79e737f/archive/2023-06-28-DssSpell/DssSpell.sol#L138)